Gemfile

```diff
@@ -49,6 +49,7 @@ gem 'simplecov', require: false, group: :test
 gem 'sinatra'
 gem 'surveyor'
 gem 'therubyracer', '0.10.2', :platforms => :ruby
+gem 'traceroute'
```

Comment:
I think this should be in the development gem block as it is not needed in prodcution